core/src/init/tag/tag.ts

```diff
@@ -0,0 +1,3 @@
+import { Tag, categoryColors } from "@moai/core";
+
+Tag.colors = categoryColors;
```

Comment:
Yeah.. well, as you can see in the preview, the PR is correct (the colors of tags are changed). However, this file is unnecessary. Tag uses "categoryColors", so updating the category colors would be enough.

You can actually see you won't need this file because this is a new file, and there is no reference to this file in "init/init.ts", but the colors do change! So basically we don't need this file :D